pay-api/src/pay_api/services/base_payment_system.py

```diff
@@ -355,9 +354,8 @@ def skip_invoice_for_sandbox(function):
     @functools.wraps(function)
     def wrapper(*func_args, **func_kwargs):
         """Complete any post invoice activities if needed."""
-        user: UserContext = func_kwargs["user"]
-        if user.is_sandbox():
```

Comment:
This isn't very useful when users use the GUI to create accounts or invoices etc, so we need to move it to an environment name check